packages/flutterfire_ui/pubspec.yaml

```diff
@@ -32,9 +32,12 @@ dependencies:
   twitter_login: ^4.2.2
 
 dev_dependencies:
+  fake_cloud_firestore: ^1.2.2
```

Comment:
Could you remove this dependency?

Reply:
This is needed in the tests.

Reply:
Right, but could you refactor the tests to remove the need for it?